### DISCUSSION TITLE
doris Can't open lib '/usr/lib64/libmyodbc5.so' : file not found (0)

### DISCUSSION THREAD

**Discussion by ghost** [apache/doris#7359] [A - General / Q&A]
使用Doris ODBC MySQL表进行查询时，当查询频繁（1秒内多次）会有（大）概率出现：
- doris Can't open lib '/usr/lib64/libmyodbc5.so' : file not found (0)
错误，具体表现为，一秒内查询三次，有一次报错。

**Comment by dataalive** [comment]
这个问题还在吗？

**Reply by hf200012** [reply]
mysql odbc driver installation problem
Please find your installation path
